molecule/default/tasks/taint.yml

```diff
@@ -315,6 +315,9 @@
       taints: "{{ taint_patch_1 }}"
     register: _result
 
+  - debug:
```

Comment:
Do we want to keep this debug here? Other than that, LGTM!

Reply:
Thanks. removed.